### PR TITLE
add date into chip

### PR DESCRIPTION
**Pull request by Le-Caignec**

### CODE REVIEW

File: src/utils/utils.ts [hunk 1]
```diff
@@ -51,7 +46,18 @@ export const createArrayBufferFromFile = async (
...
+export const getLocalDateFromTimeStamp = (
+  timestamp: TimeStamp | number
+): string => {
   const date = new Date(timestamp);
+  console.log(date);
+  return date.toLocaleDateString();
+};
+
+// from number => 1687528305
+export const getLocalDateFromNumber = (timestamp: number): string => {
```
Comment (PierreJeanjacquot) [inline src/utils/utils.ts:59]: maybe rename `getLocalDateFromNumber` into `getLocalDateFromBlockchainTimestamp`

File: src/features/protectedData/ProtectedData.tsx [hunk 2]
```diff
@@ -54,15 +55,18 @@ export default function ProtectedData() {
...
-                    id={address}
-                    title={name || 'Undefined'}
-                    schema={schema}
-                  />
-                </Grid>
-              ))}
+              {currentData?.map(
+                ({ address, name, schema, creationTimestamp }) => (
+                  <Grid item key={address}>
+                    <ProtectedDataCard
```
Comment (PierreJeanjacquot) [inline src/features/protectedData/ProtectedData.tsx:61]: some dates don't fit into the chip, fix style of `ProtectedDataCard`
image

**Review by PierreJeanjacquot — CHANGES_REQUESTED** [review]

File: src/utils/utils.ts (hunk 1 above)
Comment (Le-Caignec) [reply src/utils/utils.ts]: Commit: 022a266104e0c0cdf1863ea493fc8cb79766c4dd

File: src/features/protectedData/ProtectedData.tsx (hunk 2 above)
Comment (Le-Caignec) [reply src/features/protectedData/ProtectedData.tsx:61]: Capture d’écran 2023-06-26 à 17 25 26

**Review by PierreJeanjacquot — APPROVED** [review]
LGTM

**Pull request merged**